### PR TITLE
style(*): Fix golint issues

### PR DESCRIPTION
**Pull request by HaibinZhang-Vungle**
# Context

Fix golint issues on vungo/vast and vungo/internal packages.

# Acceptance

Golint command show no tips;
No effect to any current service.

# Changes
Code styles in vungo/vast and vungo/internal packages, including variables definition, comments formats, error string styles, and so on. Some redundant type warnings are fixed too.

### CODE REVIEW

File: internal/util/utiltest/mockcopiable.go [hunk 1]
```diff
@@ -2,14 +2,17 @@ package utiltest
 
 import "github.com/Vungle/vungo/internal/util"
 
+// MockCopiable mock file for testing copiable structs
```
Comment (yingchen0706v) [inline internal/util/utiltest/mockcopiable.go:5]: please add period in the end of the comment.

File: vast/defaults/defaults.go [hunk 2]
```diff
@@ -2,13 +2,23 @@ package defaults
 
 import "time"
 
-const MAX_VIDEO_DURATION = 60 * time.Second
-const MIN_VIDEO_DURATION = 1 * time.Second
+// MaxVideoDuration stand for the max video duration with a default of 60 seconds
```
Comment (yingchen0706v) [inline vast/defaults/defaults.go:5]: stand->stands, same for others.

File: vast/vasttest/vasttest.go [hunk 3]
```diff
@@ -81,15 +81,15 @@ func verifyElementName(t testing.TB, name string, xmlData []byte) {
 	}
 }
 
-// verifyNonEmptyXmlFields method verifies that all the fields defined in the model type has
+// verifyNonEmptyXMLFields method verifies that all the fields defined in the model type has
```
Comment (yingchen0706v) [inline vast/vasttest/vasttest.go:84]: verifyNonEmptyXMLFields method->verifyNonEmptyXMLFields function, same for others.

File: internal/util/utiltest/mockcopiable.go (hunk 1 above)
Comment (HaibinZhang-Vungle) [reply internal/util/utiltest/mockcopiable.go:5]: Add period to the end of one comment. Check other files and append a period too.

File: vast/defaults/defaults.go (hunk 2 above)
Comment (HaibinZhang-Vungle) [reply vast/defaults/defaults.go]: Replace 'stand' with 'stands' to fix this grammar error.

File: vast/vasttest/vasttest.go (hunk 3 above)
Comment (HaibinZhang-Vungle) [reply vast/vasttest/vasttest.go]: Replace 'method with 'function' to make the description more accurate.

**Review by yingchen0706v — APPROVED** [review]

**Pull request merged**